examples/fuse_examples/imaging/classification/cmmd/conf/config.yaml

```diff
@@ -5,32 +5,29 @@ paths:
   cache_dir : ${hydra:runtime.cwd}/examples/CMMD_cache_dir
   data_misc_dir : ${hydra:runtime.cwd}/data_misc
   data_split_filename:  cmmd_split.pkl
+  data_dir : ${oc.env:CMMD_DATA_PATH}
+run: 
+  running_modes : ['train', 'infer', 'eval']
 train:
   force_reset_model_dir: True
   target : classification
   reset_cache: False
-  num_workers : 12
+  num_workers : 1
```

Comment:
1 is still multiprocessing (0 is not) - so it means that multiprocessing works for you.
Why did you changed it?

Reply:
was trying to debug